bundler.go

```diff
@@ -536,6 +542,12 @@ func (b *Bundler) finishDarwin(environmentPath, binaryPath string) (err error) {
 	// Add Info.plist file
 	var fp = filepath.Join(contentsPath, "Info.plist")
 	astilog.Debugf("Adding Info.plist to %s", fp)
+	var hideLSUIElementAsString string
```

Comment:
Could you rename the var to `lsuiElement`, set it to `NO` by default and then set it to `YES` in the if condition?